language-support/hs/bindings/export-package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -eou pipefail
+
+if [ "$#" -ne 1 ]; then
+    echo "Expected exactly one argument."
+    echo "Usage: ./build_packages.sh TARGET_DIR"
```

Comment:
```suggestion
    echo "Usage: ./export-package.sh TARGET_DIR"
```
Or even better use `${BASH_SOURCE[0]}`

Reply:
Good catch! I changed to use `${BASH_SOURCE[0]}` as you suggested.